README.md

```diff
@@ -1,23 +1,25 @@
 # Packer
 
 [![Build Status][circleci-badge]][circleci]
-[![Windows Build Status][appveyor-badge]][appveyor]
+[![Discuss](https://img.shields.io/badge/discuss-packer-3d89ff?style=flat)](https://discuss.hashicorp.com/c/packer)
 [![PkgGoDev](https://pkg.go.dev/badge/github.com/hashicorp/packer)](https://pkg.go.dev/github.com/hashicorp/packer)
 [![GoReportCard][report-badge]][report]
 [![codecov](https://codecov.io/gh/hashicorp/packer/branch/master/graph/badge.svg)](https://codecov.io/gh/hashicorp/packer)
 
 [circleci-badge]: https://circleci.com/gh/hashicorp/packer.svg?style=svg
 [circleci]: https://app.circleci.com/pipelines/github/hashicorp/packer
 [appveyor-badge]: https://ci.appveyor.com/api/projects/status/miavlgnp989e5obc/branch/master?svg=true
-[appveyor]: https://ci.appveyor.com/project/hashicorp/packer
+[appveyor]: https://ci.appveyor.com/project/hashicorp/packer 
```

Comment:
Eh, actually, we don't build on appveyor anymore !

```suggestion
```

Reply:
I removed that one but forgot to remove from there! 
I will update more stuff on the readme like the JSON example and remove this one

Reply:
( ah didn't see the wip ! Will let you be :D )

Reply:
I just added the WIP because I realized it has more things to update than that 😆